### PR TITLE
[Sprint Maintain] Edit Notification & Change Domain & Update Status Shop Manangement

### PR DESCRIPTION
**Pull request by Tuanlvt**
[Sprint Maintain] Edit Notification & Change Domain & Update Status Shop Manangement
Link redmine : https://dev.framgia.com/redmine/issues/126102
Task : Maintain
Effect: N/A

### CODE REVIEW

File: app/src/main/java/com/framgia/forder/data/source/local/MyFirebaseMessagingService.java [hunk 1]
```diff
@@ -13,40 +14,49 @@
...
+        }
     }
 
     private void sendNotification(String messageBody) {
         Intent intent = new Intent(this, MainActivity.class);
         intent.addFlags(Intent.FLAG_ACTIVITY_CLEAR_TOP);
         PendingIntent pendingIntent =
-                PendingIntent.getActivity(this, DEFAULT_VALUE, intent, PendingIntent.FLAG_ONE_SHOT);
-
+                PendingIntent.getActivity(this, 0, intent, PendingIntent.FLAG_UPDATE_CURRENT);
```
Comment (nguyenhuy95dn) [inline app/src/main/java/com/framgia/forder/data/source/local/MyFirebaseMessagingService.java]: hardcode

File: app/src/main/java/com/framgia/forder/data/source/local/MyFirebaseMessagingService.java [hunk 2]
```diff
@@ -13,40 +14,49 @@
...
-                        .setContentIntent(pendingIntent);
-
+                        .setSound(defaultSoundUri);
+        notificationBuilder.setContentIntent(pendingIntent);
         NotificationManager notificationManager =
                 (NotificationManager) getSystemService(Context.NOTIFICATION_SERVICE);
-        notificationManager.notify(DEFAULT_VALUE, notificationBuilder.build());
+        SharedPreferences sharedPreferences =
+                getSharedPreferences(MainActivity.class.getSimpleName(), Context.MODE_PRIVATE);
+        int notificationNumber = sharedPreferences.getInt(PREF_NOTIFICATION_NUMBER, 0);
```
Comment (nguyenhuy95dn) [inline app/src/main/java/com/framgia/forder/data/source/local/MyFirebaseMessagingService.java]: hardcode

File: app/src/main/java/com/framgia/forder/data/source/remote/api/service/FOrderApi.java [hunk 3]
```diff
@@ -204,6 +204,9 @@
             @Query("domain_id") int domainId, @Query("user_id") int userId,
             @Query("role") String role);
 
+    @GET("v1/dashboard/shops/{shop_id}/edit")
+    Observable<BaseResponse> requestChangeStatusShopManagement(@Query("shop_id") int shopId);
```
Comment (nguyenhuy95dn) [inline app/src/main/java/com/framgia/forder/data/source/remote/api/service/FOrderApi.java]: @Query -> @Path

File: app/src/main/java/com/framgia/forder/data/source/local/MyFirebaseMessagingService.java [hunk 4]
```diff
@@ -21,32 +22,43 @@
...
                 (NotificationManager) getSystemService(Context.NOTIFICATION_SERVICE);
-        notificationManager.notify(DEFAULT_VALUE, notificationBuilder.build());
+        SharedPreferences sharedPreferences =
+                getSharedPreferences(MainActivity.class.getSimpleName(), Context.MODE_PRIVATE);
+        int notificationNumber = sharedPreferences.getInt(PREF_NOTIFICATION_NUMBER, DEFAULT_VALUE);
+        notificationManager.notify(notificationNumber, notificationBuilder.build());
+        SharedPreferences.Editor editor = sharedPreferences.edit();
+        notificationNumber++;
+        editor.putInt(PREF_NOTIFICATION_NUMBER, notificationNumber);
+        editor.apply();
```
Comment (daolq3012) [inline app/src/main/java/com/framgia/forder/data/source/local/MyFirebaseMessagingService.java]: where is clear ``notificationNumber`` value from SharedPreferences?

**Pull request merged**